src/renderDigging.js

```diff
@@ -423,14 +423,15 @@ function getPipPoints(state, row, column) {
     const x = column * COLUMN_WIDTH - state.camera.left;
     const y = row * ROW_HEIGHT - state.camera.top + ((column % 2) ? LONG_EDGE : 0);
     const pad = 5;
+    const c = 0.4;
     return [
-        [x + HEX_WIDTH / 2, y + ROW_HEIGHT / 6 + pad * 1.5],
-        [x + SHORT_EDGE + pad, y + ROW_HEIGHT / 3 + pad],
-        [x + HEX_WIDTH - SHORT_EDGE - pad, y + ROW_HEIGHT / 3 + pad],
-        [x + HEX_WIDTH / 2, y + ROW_HEIGHT / 2],
-        [x + SHORT_EDGE + pad, y + 2 * ROW_HEIGHT / 3 - pad],
-        [x + HEX_WIDTH - SHORT_EDGE - pad, y + 2 * ROW_HEIGHT / 3 - pad],
-        [x + HEX_WIDTH / 2, y + 5 * ROW_HEIGHT / 6 - pad * 1.5],
+        [x + HEX_WIDTH / 4 + HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + 3 * HEX_WIDTH / 4 - HEX_WIDTH / 4 * c, y + ROW_HEIGHT / 2 * c],
+        [x + SHORT_EDGE * c, y + ROW_HEIGHT / 2 * c],
+        [x + HEX_WIDTH * c/ 2, y + ROW_HEIGHT / 2 * c],
```

Comment:
This line shouldn't have `c`, this is the center dot, and is always in the same position.

Reply:
Also, did you want to add those screenshots, or should I just go ahead and merge this in?

Reply:
oh sorry, I thought you meant on future merges do screenshots.
And I did it right on the Visual Code I made the change separately on Github and should've just copy and pasted.